kyo-core/shared/src/main/scala/kyo/Async.scala

```diff
@@ -39,6 +39,11 @@ object Async:
       *   A Fiber representing the running computation
       */
     inline def run[E, A: Flat, Ctx](inline v: => A < (Abort[E] & Async & Ctx))(
+        using frame: Frame
+    ): Fiber[E, A] < (IO & Ctx) =
+        _run(v)
+
+    private[kyo] inline def _run[E, A: Flat, Ctx](inline v: => A < (Abort[E] & Async & Ctx))(
```

Comment:
As discussed offline, I don't think these can be `private` at all due to the `inline`... We may need to find a way to test only public `inline` functions in another namespace...

Reply:
I've just checked by moving `AsyncTest` to a `kyoTest` package and it seems ok.